bind/symtab.go

```diff
@@ -130,6 +130,10 @@ func (s symbol) isMap() bool {
 	return (s.kind & skMap) != 0
 }
 
+func (s symbol) isSequence() bool {
```

Comment:
> a quick grep in the code seems to indicate isSequence is used to determine whether we need to generate methods that implement the python's sequence protocol...
Yes, that is. :)
> in other words, do we want to also test for maps?
Yes, we need to generate protocols for map also. I am investigating about that.